packages/js-moi-manifest/src.ts/element-descriptor.ts

```diff
@@ -41,6 +57,10 @@ export default class ElementDescriptor {
                     };
                     this.callSites.set(routineData.name, callsite);
                     break;
+                case "event":
+                    const eventData = element.data as LogicManifest.Event;
+                    this.eventsDef.set(eventData.name, { ptr: element.ptr, topics: eventData.topics });
```

Comment:
nitpick, make it `eventDefs`.

Reply:
https://github.com/sarvalabs/js-moi-sdk/pull/88/commits/44e0136ec5567014b65e346ea67b8e35b2858344